json_schemas/brief-awards-digital-outcomes-and-specialists-2-digital-outcomes.json

```diff
@@ -7,7 +7,7 @@
       "type": "string"
     },
     "awardedContractValue": {
-      "pattern": "^\\d{1,15}(?:\\.\\d{1,5})?$",
+      "pattern": "^[1-9](?:\\d{1,15})?(?:\\.\\d{2})?$|^0\\.(?!00)\\d{2}$",
```

Comment:
Same comment re: pattern; should probably be `{1,14}`.

Reply:
Have updated